step-capstone/src/components/TripSettingFormPopover.js

```diff
@@ -91,62 +94,74 @@ function EditTripSetting(props) {
 
     useEffect(() => {
         props.onDataChange({
+            title: title,
             destinations : destinations,
             startDate : startDate,
             endDate : endDate,
-            description :  description
+            description :  description,
         })
         // notifies form if necessary inputs are present
-        props.onValidation(!(destinations === ""))
+        props.onValidation(!(destinations === "" || (title === "")))
```

Comment:
do we need them to list destinations?

Reply:
as I said above, this part will be handled by Emmie's code.